integration-cli/docker_cli_build_test.go

```diff
@@ -5655,8 +5648,13 @@ func (s *DockerSuite) TestBuildRUNErrMsg(c *check.C) {
 	if err == nil {
 		c.Fatal("Should have failed to build")
 	}
-
-	exp := `The command '/bin/sh -c badEXE a1 \& a2	a3' returned a non-zero code: 127`
+	shell := "/bin/sh -c"
+	exitCode := "127"
+	if daemonPlatform == "windows" {
+		shell = "cmd /S /C"
+		exitCode = "1"
```

Comment:
I would add a comment here explaining the difference (architectural - Windows has to start the container to determine the exe is bad, Linux does not)